server/db.js

```diff
@@ -1,112 +1,184 @@
-const Sequelize = require('sequelize');
-// const Seed = require('../seed.js');
-const { username, password } = require('../config.js');
+const { Pool, Client } = require('pg');
 
+const pool = new Pool({
 
-const sequelize = new Sequelize('player', username, password, {
-  host: 'localhost',
-  dialect: 'mysql',
-});
+  user: 'moo',
```

Comment:
Consider using a config file because not everybody will have these configurations, and if you had a password, you wouldn't want it to be on github. Make sure to update your README with config instructions as well.

Reply:
Done!